WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -204,11 +204,12 @@ private extension TotalsView {
             .frame(minWidth: UIScreen.main.bounds.width / 2)
         })
         .padding(Constants.newOrderButtonPadding)
-        .foregroundColor(Color.posPrimaryText)
+        .foregroundColor(.posPrimaryTextInverted)
```

Comment:
I thought about creating a new Button style for this change, but since we only use it once across all flows I opted for the simplest change of creating a new colour.